corehq/apps/app_manager/xform.py

```diff
@@ -2133,7 +2133,7 @@ def infer_vellum_type(control, bind):
 
 def find_missing_instances(form_source):
     from corehq.apps.app_manager.suite_xml.post_process.instances import get_all_instances_referenced_in_xpaths
-    instance_declaration_re = r"""<instance.*id=\"([\w\-:]+)\""""
+    instance_declaration_re = r"""<instance[^>]*id=\"([\w\-:]+)\""""
```

Comment:
I think this will miss an instance formatted like `<instance id='casedb' src='...'>` (single quotes). Is that a concern here?

Also, less important, this will match tags like `<instances id="xyz"` and `<instanceid="xyz"`.

A better regexp might be `<instance\s[^>]*id=(['"])([\w\-:]+)\1` – note that this one has different group indices, so you may need to account for that elsewhere in the code. Tested on the following:
```xml
<instance src="..." id="abc">
<instance src="..." id='abc'>
<instanceid="abc">
<instances
  src="..." id="abc">
<instance
  src="..." id="abc">
```